spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -26,7 +26,7 @@ An observer method will be notified of an event if the event object is assignabl
 An event object is an instance of a concrete Java class with no unresolvable type variables.
 The _event types_ of the event include all superclasses and interfaces of the runtime class of the event object.
 
-An event type may not contain an unresolvable type variable.
+An event type may not contain an unresolvable type variable. A wildcard type is not considered an unresolvable type variable.
```

Comment:
Be careful here. In JLS Wildcard type are unbound (<?>) or bound (<? extends Number>) are you sure you mean both here?